src/containers/PermissionsApplication.jsx

```diff
@@ -29,7 +29,21 @@ const PermissionsApplication = ({ match }) => {
           {t('Permissions.failedRequest')}
         </Typography>
       ) : (
-        <PageTitle>{appName}</PageTitle>
+        <div>
+          <PageTitle>{appName.toUpperCase()}</PageTitle>
+          {Object.entries(queryResult.data[0].attributes.permissions).map(
+            ([key, value]) => (
+              <React.Fragment key={key}>
+                <Typography variant="h4">{key}</Typography>
+                {value.verbs && (
```

Comment:
can we add inside the snapshot, an App, with a value containing verbs please?

Reply:
@Noemie21 : Sometimes you'll not have any verbs. When not having verbs it means that you have all the rights. This is the same as "ALL" or the same as "GET / PUT / POST / DELETE"  

Ex: https://github.com/cozy/cozy-notes/blob/master/manifest.webapp#L43-L46

Reply:
Also, I think that the important stuff is to displayed the concerned doctype (aka the type) instead of the `key`.

I would prefer to see: 
io.cozy.files: ALL 
io.cozy.contacts: GET 

instead of: 
allFiles: ALL
Contact: GET